rsmtool/notebooks/evaluation.ipynb

```diff
@@ -133,10 +151,13 @@
    "outputs": [],
    "source": [
     "markdown_strs = [\"The histogram and the table below show the distibution of \"\n",
-    "                 \"human scores and {} trimmed rounded machine scores \"\n",
+    "                 \"rounded human scores and {} trimmed rounded machine scores \"\n",
     "                 \"(as % of all responses).\".format(raw_or_scaled)]\n",
     "markdown_strs.append(\"Differences in the table between human and machine distributions \"\n",
     "                     \"larger than 5 percentage points are <span class='highlight_color'>highlighted</span>.\")\n",
+    "if continuous_human_score:\n",
+    "    markdown_str.append(\"*Human scores have beeen rounded to the nearest integer.*\")\n",
```

Comment:
Shouldn't I see this text in the `lr-with-continuous-human-scores` report? I don't. Ah, there's a typo `markdown_str` should be `markdown_strs` with an s. Also, I would remove the italics here and start with "Note that ..."

Reply:
Good call! I thought I checked them all but I missed this one!